tests/NEST.py

```diff
@@ -0,0 +1,659 @@
+from nestpy import nestpy
+from types import *
+import unittest
+from enum import Enum
+
+import sys
+
+def trace(frame, event, arg):
+    print "%s, %s:%d" % (event, frame.f_code.co_filename, frame.f_lineno)
```

Comment:
Can you use print() with the brackets so it works in Python3?

Reply:
Removed